Tests/DependencyInjection/SwarrotExtensionTest.php

```diff
@@ -90,6 +90,18 @@ public function test_it_registers_the_collector_when_explicitly_enabled()
         $this->assertHasService($container, 'swarrot.data_collector');
     }
 
+    public function test_it_use_the_asked_logger()
+    {
+        $container = $this->createContainer(false);
+
+        $this->loadConfig($container, array('publisher_logger' => 'my_awesome_logger'));
```

Comment:
As this test uses the deprecated config, you should mark it as ``@group legacy`` to be compatible with the usage of the PHPUnit bridge to handle usage of deprecated APIs in tests (and you should add the bridge as a dev requirement if it is not there yet IMO)